test/cmd/packet-verifier/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG builder_image=amazonlinux:2
+ 
+FROM $builder_image as builder
+ENV GO111MODULE=on
+ENV GOPROXY=direct
+ENV GOOS=linux
+ENV GOARCH=amd64
+COPY . /
+WORKDIR /
+RUN yum install -y git golang libpcap-devel
+RUN go build -o packet-verifier packet-verifier.go
+ 
+FROM amazonlinux:2
+RUN yum install -y libpcap-devel
```

Comment:
Do we need all of `libpcap-devel` here, or will just the runtime libraries in `libpcap` do?

Reply:
I initially tried with just libpcap but that didn't help. Had to pull in couple of dependency from libpcap-devel as well.